operator/mapper/dbaas.go

```diff
@@ -50,3 +51,19 @@ func CompareSettings(a, b runtime.RawExtension) bool {
 	}
 	return true
 }
+
+func CompareMajorVersion(a, b string) (bool, error) {
+	va, err := version.NewVersion(a)
```

Comment:
Maybe add a bit of godoc? does it matter if there's a `v` prefix or not?

Reply:
please, let me know if that's not enough